src/main/app/fileUpload/documentManager.ts

```diff
@@ -2,16 +2,19 @@
 
 import axios, { AxiosInstance, RawAxiosRequestHeaders } from 'axios';
 import config from 'config';
+import { Logger } from '../../../test/unit/mocks/hmcts/nodejs-logging';
```

Comment:
Tnis is a test mock logger, import the one below

import { Logger } from '@hmcts/nodejs-logging';